examples/entity_linking.py

```diff
@@ -0,0 +1,33 @@
+from wellcomeml.ml import EntityLinker
+
+entities_kb = {
+        'Michelle Williams (actor)': "American actress. She is the recipient of several accolades, including two Golden Globe Awards and a Primetime Emmy Award, in addition to nominations for four Academy Awards and one Tony Award.",
+        'Michelle Williams (musician)': "American entertainer. She rose to fame in the 2000s as a member of R&B girl group Destiny's Child, one of the best-selling female groups of all time with over 60 million records, of which more than 35 million copies sold with the trio lineup with Williams.",
```

Comment:
this representation might be a bit limited. how will you construct a unique id? will you keep the id separately? i.e. Jeremmy Farrar might not be unique, if you replace with ORCID id where will you keep the association between ORCID and name? it wold be nice if the knowledge base contained this information, essentially the value was not text but a dict that contained text along with other information

Reply:
The `entities_kb` dictionary key is the unique id, and can be anything as long as it is consistent with what is used in the tag (e.g. `{'id': 'Michelle Williams (musician)'}`), so this could be the ORCID or another ID you want to use. I just put their names so the print out would be clearer.

In terms of using ORCIDs I wouldn't have 'Jeremy Farrar', I'd have his ORCID.

If you think this point isn't clear in this example I can change it to:
```
entities_kb = {
        'id_1': "American actress. She ....",
        'id_2': "American entertainer. She....",
        'id_3': "  "
        }
train_data = [
        ("After Destiny's ...", {'id': 'id_2'}),
        ("On Broadway ....", {'id': 'id_1'}),
        ("Franklin would ...", {'id': 'No ID'})
        ]
```